perftests/standalone-enc/fd_drain.h

```diff
@@ -0,0 +1,23 @@
+#ifndef FD_DRAIN_H_
+#define FD_DRAIN_H_
+
+#include <sys/types.h>
+
+/* The smallest this can be is PCRYPT_ESZ (which is 1060). */
```

Comment:
Not sure if this comment is necessary or even correct any more?

Reply:
Hmm, I guess that if it's something tiny like 10 bytes, that would just mean `read` gets called more often.

... yeah, I'll delete it.